src/webui/webapplication.cpp

```diff
@@ -428,6 +424,10 @@ void WebApplication::configure()
     if (m_currentLocale != newLocale) {
         m_currentLocale = newLocale;
         m_translatedFiles.clear();
+        if (m_translator.load(m_rootFolder + QLatin1String("/translations/webui_") + m_currentLocale))
+            LogMsg(tr("WebUI %1 locale recognized, using translation.").arg(m_currentLocale));
```

Comment:
Minor thing, I would use "WebUI switching to locale: %1."
And for below: "WebUI invalid locale selected: %1, falling back to default: en"

Reply:
I just use existing messages from Application class.

>WebUI switching to locale
WebUI invalid locale selected

IMO, it's incorrect. The event is "Translation for selected locale loaded or failed to load".
So how about "WebUI translation for selected locale (%1) successfully loaded." and "Couldn't load WebUI translation for selected locale (%1), falling back to default (en)."?

Reply:
ok.